js/views/pageNavVw.js

```diff
@@ -591,14 +611,175 @@ module.exports = baseVw.extend({
       Backbone.history.navigate(route, {trigger: true});
     });
   },
+  
```

Comment:
Problem with such approach is that suggestionListView wouldn't be able to access the search input the backbone way - this.$('.element'). I've seen accessing various elements from non-related views though, so I guess we don't care about it that much? 

If you, on the other hand, put the input to suggestionsList.html then you'd have to move all the addressBar*() methods from pageNavView to the suggestionsListView and have a problem accessing closeStatusBar() that is called in addressBarKeyup(), and also called from within pageNavView numerous times.

At first, I made a separate view for suggestions and then I stumbled upon above issues and couldn't find a "pretty" way to solve it. That's why everything ended up in pageNavView. I know it isn't ideal.